### PR TITLE
feat(dedicated): private cloud service pack enhancements

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-5491
| License          | BSD 3-Clause

## Description

Enhance service packs management in a private cloud

### CODE REVIEW

**Review by marie-j — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html [hunk 1]
```diff
@@ -1,75 +1,47 @@
...
->
-    <div
-        class="orderSelection__product-container"
-        data-ng-if=":: $ctrl.servicePacks[i]"
-        data-ng-repeat="i in ::[$index, $index + 1]"
+<div class="row mb-3">
+    <oui-select-picker
+        class="col-md-4 mb-3"
+        data-ng-repeat="servicePack in ::$ctrl.servicePacks track by servicePack.name"
+        name="servicePack"
```
Comment (ganeshkumar1989) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html:5]:
```suggestion
        data-name="servicePack"
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html [hunk 2]
```diff
@@ -1,75 +1,47 @@
...
+                class="dedicatedCloud-servicePack-option-description"
+                data-translate="{{:: servicePack.description }}"
+            ></span>
+        </oui-select-picker-section>
+        <oui-select-picker-footer>
+            <span
+                class="d-block"
+                data-ng-if="servicePack.prices.monthly.exists"
+                data-translate="dedicatedCloudDashboardTilesOptionsOrderSelection_priceDifferenceOnMonthlyBill"
+                data-translate-values=":: {
```
Comment (ganeshkumar1989) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html:31]:
```suggestion
                data-translate-values="{
```

Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html:31]: I guess data-translate-values is already single binded...

**Review by ganeshkumar1989 — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html (hunk 1 above)
Comment (radireddy) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html:5]: name is a standard HTML attribute. date- is not required.

**Comment by ovh-cds** [comment]
CDS Report translator-push#10583.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#10584.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#10585.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#10586.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#10587.0 ✘  
* 
  * push ✘

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html (hunk 2 above)
Comment (radireddy) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/components/picker/service-pack-picker.html:31]: yes but I think it's ok to have it

**Review by marie-j — APPROVED** [review]

**Pull request merged**